moveit_runtime/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package>
+  <name>moveit_runtime</name>
+  <version>0.7.6</version>
```

Comment:
Upon release, the version needs to be aligned with other packages in this repo for each distro (the version here falsely remains as the one from indigo).